app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelContent.tsx

```diff
@@ -33,13 +33,13 @@ const ConnectionPanelContent = ({
 	contentHeader,
 	contentInstructionOne,
 	contentInstructionTwo,
-	buttonAppearance,
+	buttonOneAppearance,
 	firstButtonLabel,
 	secondButtonLabel,
 	buttonOneOnClick,
 	buttonTwoOnClick,
-	testId,
-	isIph,
+	buttonOneTestId,
+	isButtonOneIph,
```

Comment:
/nit - You are being consistent here with other components, which is good, but I have to remember everytime i see IPH what i am looking at
i would lean toward **isButtonOneIph** -> **isButtonOneInProductHelp**